vault/seal_util.go

```diff
@@ -121,3 +121,42 @@ func UnsealWrapStoredBarrierKeys(ctx context.Context, access seal.Access, pe *ph
 	}
 	return keys, nil
 }
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Recovery Key
+
+// SealWrapRecoveryKey encrypts the recovery key using the given seal access and returns a physical.Entry for storage.
+func SealWrapRecoveryKey(ctx context.Context, access seal.Access, key []byte) (*physical.Entry, error) {
+	blobInfo, err := SealWrapValue(ctx, access, true, key)
+	if err != nil {
+		return nil, &ErrEncrypt{Err: fmt.Errorf("failed to encrypt keys for storage: %w", err)}
```

Comment:
nit: can you change this to "failed to encrypt recovery keys for storage"

Reply:
Sorry Divya, I didn't make this change on this PR, but I will change the message and include the change in my next PR.